tests/unit/helper.py

```diff
@@ -52,6 +52,14 @@ class UnitHelper(unittest.TestCase):
     # Sub-classes must also assign a dictionary to this during definition
     example_data = {}
 
+    def assert_requires_auth(self, func):
```

Comment:
Could you add this to `UnitRequiresAuthHelper` ? I thought I had converted all the requires auth classes to that helper. I'm very +1 on this method though.